### PR TITLE
refactor(dms/datasource): fix lint issues

### PR DESCRIPTION
**Pull request by chengxiangdong**
**What this PR does / why we need it**:

**Which issue this PR fixes**:
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
fixes #xxx

**Special notes for your reviewer**:

- Fix some lint issues in `huaweicloud_dms_maintainwindow` and `huaweicloud_dms_product`.

**Release note**:

```release-note
NONE
```

## PR Checklist

* [x] Tests added/passed.
* [ ] Documentation updated.
* [ ] Schema updated.

## Acceptance Steps Performed

```
make testacc TEST='./huaweicloud/services/acceptance/dms' TESTARGS='-run TestAccDmsMaintainWindowDataSource'
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test ./huaweicloud/services/acceptance/dms -v -run TestAccDmsMaintainWindowDataSource -timeout 360m -parallel 4
=== RUN   TestAccDmsMaintainWindowDataSource_basic
=== PAUSE TestAccDmsMaintainWindowDataSource_basic
=== CONT  TestAccDmsMaintainWindowDataSource_basic
--- PASS: TestAccDmsMaintainWindowDataSource_basic (12.84s)
PASS
ok      github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/acceptance/dms       12.947s

```

```
make testacc TEST='./huaweicloud/services/acceptance/dms' TESTARGS='-run TestAccDmsProductDataSource'
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test ./huaweicloud/services/acceptance/dms -v -run TestAccDmsProductDataSource -timeout 360m -parallel 4
=== RUN   TestAccDmsProductDataSource_basic
=== PAUSE TestAccDmsProductDataSource_basic
=== RUN   TestAccDmsProductDataSource_kafkaVmSpec
=== PAUSE TestAccDmsProductDataSource_kafkaVmSpec
=== RUN   TestAccDmsProductDataSource_rabbitmqSingle
=== PAUSE TestAccDmsProductDataSource_rabbitmqSingle
=== RUN   TestAccDmsProductDataSource_rabbitmqCluster
=== PAUSE TestAccDmsProductDataSource_rabbitmqCluster
=== CONT  TestAccDmsProductDataSource_basic
=== CONT  TestAccDmsProductDataSource_rabbitmqSingle
=== CONT  TestAccDmsProductDataSource_kafkaVmSpec
=== CONT  TestAccDmsProductDataSource_rabbitmqCluster
--- PASS: TestAccDmsProductDataSource_rabbitmqSingle (21.72s)
--- PASS: TestAccDmsProductDataSource_basic (23.58s)
--- PASS: TestAccDmsProductDataSource_rabbitmqCluster (23.59s)
--- PASS: TestAccDmsProductDataSource_kafkaVmSpec (24.21s)
PASS
ok      github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/acceptance/dms       24.336s

```

### CODE REVIEW

**Comment by chengxiangdong** [comment]
All work done and ready for review.

**Comment by chengxiangdong** [comment]
All work done and ready for review.
@ShiChangkuo

**Comment by chengxiangdong** [comment]
All work done and ready for review.

File: huaweicloud/services/dms/data_source_huaweicloud_dms_maintainwindow.go [hunk 1]
```diff
@@ -2,19 +2,17 @@ package dms
...
 
 	"github.com/hashicorp/go-multierror"
 	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
 
-	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/utils"
-	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/utils/fmtp"
-	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/utils/logp"
-
 	"github.com/chnsz/golangsdk/openstack/dms/v2/maintainwindows"
 	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
```
Comment (houpeng80) [inline huaweicloud/services/dms/data_source_huaweicloud_dms_maintainwindow.go]: it is better to put the package of hashicorp together

File: huaweicloud/services/dms/data_source_huaweicloud_dms_product.go [hunk 2]
```diff
@@ -2,14 +2,13 @@ package dms
...
 
 	"github.com/hashicorp/go-multierror"
 	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
 	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/validation"
 
-	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/utils/fmtp"
-	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/utils/logp"
-
 	"github.com/chnsz/golangsdk/openstack/dms/v2/products"
 	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
```
Comment (houpeng80) [inline huaweicloud/services/dms/data_source_huaweicloud_dms_product.go]: ditto

File: huaweicloud/services/dms/data_source_huaweicloud_dms_maintainwindow.go (hunk 1 above)
Comment (chengxiangdong) [reply huaweicloud/services/dms/data_source_huaweicloud_dms_maintainwindow.go]: good idea

File: huaweicloud/services/dms/data_source_huaweicloud_dms_product.go (hunk 2 above)
Comment (chengxiangdong) [reply huaweicloud/services/dms/data_source_huaweicloud_dms_product.go]: done

**Pull request merged**